### ISSUE TITLE
SmartCity-Accessibility Server Side #563 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#336]
Build 'SmartCity-Accessibility Server Side' is failing!

Failed Tests :
```

```

Changes since last successful build:
No changes
No changes

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/563/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
